libs/external/shadcn/components/ui/input.stories.tsx

```
@@ -0,0 +1,35 @@
+}
+
+const Template = (args: React.ComponentProps<typeof Input>) => <Input {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  type: 'text',
+  placeholder: 'Enter text...',
+  className: 'border-2 border-border p-2 rounded-md',
+}
+
+export const Number = Template.bind({})
```

Biome (1.9.4)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)